sure.py

```diff
@@ -423,6 +423,32 @@ def serverType():
 	elif 'Amazon'in server_type or 'Microsoft' in server_type:
 		return 'on-cloud'
 
+#Persona type: COMPUTE/DATA/COMPUTE_AND_DATA
+def personaType():
+	persona_type = re.findall('"([^"]*)"',str(executeCommand('cat /opt/web-app/etc/persona')))[1]
+	return persona_type
+
```

Comment:
This needs exception handling. In case file is not found, it will cause index exception error